sympy/vector/functions.py

```diff
@@ -1,11 +1,11 @@
 from sympy.vector.coordsysrect import CoordSys3D
-from sympy.vector.deloperator import Del
 from sympy.vector.scalar import BaseScalar
 from sympy.vector.vector import Vector, BaseVector
 from sympy.vector.operators import gradient, curl, divergence
 from sympy import diff, integrate, S, simplify
 from sympy.core import sympify
 from sympy.vector.dyadic import Dyadic
+from sympy.vector.deloperator import Del
```

Comment:
Is there a specific reason to move this? If anything the import of Dyadic should have moved up so that all imports from sympy.vector is in alphabetical order.

Reply:
No, i removed it while trying to fix some circular import issue. I looks like i did not placed it where it was before. I will correct it.